records_mover/records/targets/table/move_from_fileobjs_source.py

```diff
@@ -30,9 +30,13 @@ def __init__(self,
         super().__init__(prep, target_table_details, processing_instructions)
 
     def load(self, driver: DBDriver) -> Optional[int]:
-        return driver.load_from_fileobj(schema=self.tbl.schema_name,
-                                        table=self.tbl.table_name,
-                                        load_plan=self.plan, fileobj=self.fileobj)
+        loader_from_fileobj = driver.loader_from_fileobj()
+        # TODO: explain this one
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker